docs/4.0-migration-guide/platform-entropy-gathering.md

```diff
@@ -0,0 +1,63 @@
+## Platform-specific Entropy Gathering
+
+### Disclaimer
+
+All the following sections refer to the case where entropy module uses default
+entropy sources, i.e. `MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES` is not defined.
+
+### Background
```

Comment:
I think there's too much detail about what was there before. It's useful to have some explanations, because some readers will be programmers who have to upgrade some part of the application that was written by a long-gone colleague and that they don't understand. But we don't need to explain everything about the old API, especially the parts that haven't changed (e.g. the platform sources are exactly the same as before, what's changed is how you say whether you want to use them).